modules/backend/widgets/form/partials/_form_tabs.htm

```diff
@@ -13,20 +13,36 @@
 ?>
 <div class="<?= $navCss ?>">
     <ul class="nav nav-tabs">
-        <?php $index = 0; foreach ($tabs as $name => $fields): ?>
-            <li class="<?= $index++ == 0 ? 'active' : '' ?>">
-                <a href="#<?= $type . 'tab-' . $index ?>"><?php if ($tabs->getIcon($name)): ?><span class="<?= $tabs->getIcon($name) ?>"></span><?php endif; ?><?= e(trans($name)) ?></a>
+        <?php
+        $index = 0;
+        foreach ($tabs as $name => $fields):
+            $lazy = in_array($name, $tabs->lazy);
+        ?>
+            <li class="<?= $index++==0?'active':''?> <?= $lazy ? 'tab-lazy' : '' ?>">
```

Comment:
@tobias-kuendig Would you be able to keep the spacing here with the index ternary condition?
```suggestion
            <li class="<?= ($index++ === 0) ? 'active' : '' ?> <?= $lazy ? 'tab-lazy' : '' ?>">
```